### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by chevdor**
This PR fixes linting issues and benefit from newer syntax to simplify the code.
It is not fixing 100% of the issues as some separated topics have been splitted out in #306 and #307.

### CODE REVIEW

**Comment by chevdor** [comment]
Solving #307 would make working on this PR much easier.

**Comment by chevdor** [comment]
Rebased master.
This PR will still show linting errors as it does not include the fixes from #306 and #307

File: src/builder/import.rs [hunk 1]
```diff
@@ -17,11 +17,17 @@ impl ImportBuilder {
 	}
 }
 
+impl Default for ImportBuilder {
+	fn default() -> Self {
+		Self::new()
+	}
+}
+
```
Comment (athei) [inline src/builder/import.rs:25]: Why all those extra `Default` implementations? Is it used somewhere or is is for the create users?

**Review by athei — APPROVED** [review]
`module.rs` has mainly formatting changes which made it hard to review :(

File: src/builder/import.rs (hunk 1 above)
Comment (chevdor) [reply src/builder/import.rs:25]: This is for the crate users who are to be expected to look for a `Default` impl. if fn new()` takes no input. See https://rust-lang.github.io/rust-clippy/master/#new_without_default

**Pull request merged**